src/index.html

```diff
@@ -1,22 +1,26 @@
 <!doctype html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    />
-    <meta
-      http-equiv="X-UA-Compatible"
-      content="ie=edge"
-    />
-    <title>Moyo header</title>
-    <link
-      rel="stylesheet"
-      href="./style.css"
-    />
-  </head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <title>Moyo Header</title>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet" />
+  <link rel="stylesheet" href="./style.css" />
+</head>
   <body>
-    <h1>Moyo header</h1>
-  </body>
+  <header class="header">
+  <a href="#" class="logo">
+    <img src="src/images/logo.png" alt="Moyo company logo" class="logo__img"></a>
```

Comment:
image path is incorrect because index.html is already inside src folder so `src/images/logo.png ` points to `src/src/...`  that doesn't exist
```suggestion
    <img src="./images/logo.png" alt="Moyo company logo" class="logo__img"></a>
```
<img width="1079" height="352" alt="image" src="https://github.com/user-attachments/assets/5d2a4255-a9a1-47eb-b269-b245167a1edc" />